client/src/app/AppParent.js

```diff
@@ -397,6 +399,7 @@ export default class AppParent extends PureComponent {
         onReady={ this.handleReady }
         onError={ this.handleError }
         onWarning={ this.handleWarning }
+        hasServerInteraction={ hasServerInteraction }
```

Comment:
We do not have to pass the flags here but could also directly use them:

```
import Flags from `...`

if (Flags.get('....') {

}
```

This way it is possible to check for the flag inside the plug-in and require no knowledge of the app.

Reply:
It's not like that this is a better way to do it, just wanted to make sure you've had that one on your radar when implementing the solution.

Reply:
Thanks!